src/i18n/en.json

```diff
@@ -186,7 +186,7 @@
     "ui.package.hintConstraintUpdate": "This package will be updated when you apply the changes.",
     "ui.package.hintRemoved": "This package will be removed when you apply the changes.",
     "ui.package.requiredTitle": "manually added",
-    "ui.package.requiredText": "This package is required in your composer.json but not installed.",
+    "ui.package.requiredText": "This package is required in your <code>composer.json</code> but not installed.",
```

Comment:
For security reasons, HTML cannot be included by default in labels.

Reply:
But there are already HTML tags in other labels as well.

Reply:
Yes, in very limited cases where we must support them…

Reply:
I have seen it e.g. here:

https://github.com/xchs/contao-manager/blob/master/src/i18n/en.json#L151
https://github.com/xchs/contao-manager/blob/master/src/i18n/en.json#L158